spanner/session.go

```diff
@@ -597,9 +640,14 @@ type sessionPool struct {
 	// numInUse is the number of sessions that are currently in use (checked out
 	// from the session pool).
 	numInUse uint64
+	// multiplexedSessionInUse is the number of transactions using multiplexed sessions.
+	multiplexedSessionInUse uint64
 	// maxNumInUse is the maximum number of sessions in use concurrently in the
 	// current 10 minute interval.
 	maxNumInUse uint64
+	// maxMultiplexedSessionInUse is the maximum number of multiplexed sessions in use concurrently in the
+	// current 10 minute interval.
+	maxMultiplexedSessionInUse uint64
```

Comment:
Removed this since it is not adding any real benefit to customer other than drop in InUse and maxInUse metrics on customer when using multiplexed session.